code/modules/antagonists/disease/disease_mob.dm

```diff
@@ -76,10 +76,12 @@ the new instance inside the host to be updated to the template's stats.
 /mob/camera/disease/Destroy()
 	. = ..()
 	QDEL_NULL(adaptation_menu_action)
+	disease_template = null
 	for(var/V in GLOB.sentient_disease_instances)
 		var/datum/disease/advance/sentient_disease/S = V
 		if(S.overmind == src)
 			S.overmind = null
+	browser = null
```

Comment:
Shouldn't this get deleted. Also this needs modernization but that's another PR.

Reply:
I don't think browsers implement destroy, I'll check later

Reply:
yeah, browsers don't implement destroy, which means qdeling them will just lead to hanging refs in other bits of code. better to just byond deal with it, similarly to gasmixtures